plugins/AITagger/ai_tagger.py

```diff
@@ -0,0 +1,345 @@
+import os
+import sys
+import json
+import subprocess
+import csv
+from typing import Any
+
+# plugins don't start in the right directory, let's switch to the local directory
+os.chdir(os.path.dirname(os.path.realpath(__file__)))
+
+# ----------------- Setup -----------------
+
+def install(package):
+    subprocess.check_call([sys.executable, "-m", "pip", "install", package])
```

Comment:
will likely raise errors on docker container without --break-system-packages and bypasses venv fyi

Reply:
is there currently a better best-practice way to auto-install dependencies?

Reply:
it is a mess righ tnow, probably not a better way but including a requirements.txt I believe will trigger py_common and stash-s6

Reply:
I'll add a requirements.txt and surround this install in a try catch which gives a more user-friendly message if they try to install on docker or an environment where this wont work with a message detailing how to manually install the dependencies if all else failed.